static/templates/system/settings/homebrew.hbs

```diff
@@ -11,7 +11,7 @@
         <div class="form-group setting">
             <label for="{{options.id}}-languages">{{localize settings.languages.name}}</label>
             <div class="form-fields">
-                <input type="text" name="languages" class="homebrew pf2e-tagify" data-dtype="JSON" value="{{json traitSettings.languages.value}}">
+                <input type="text" name="languages" class="homebrew pf2e-tagify" data-dtype="JSON" value="{{json settings.languages.value}}">
```

Comment:
This is the fix, right?

Reply:
No. Its still broken even with just this change. I think something changed in core that caused the rerender of the rarity update to not reflect changes in the languages themselves.